src/main/java/org/mockito/internal/util/Platform.java

```diff
@@ -0,0 +1,48 @@
+package org.mockito.internal.util;
+
+public abstract class Platform {
```

Comment:
Yes true, but that is what the private constructor guarantees too right? But leaving it in is fine too :)